tools/osbuilder/rootfs-builder/rootfs.sh

```diff
@@ -657,9 +657,9 @@ EOF
 	touch "$dns_file"
 
     if [ -n "${AA_KBC}" ]; then
-		if [ "${AA_KBC}" == "offline_sev_kbc" ]; then
+		if [[ "${AA_KBC}" =~ ^offline_(sev|fs)_kbc$ ]]; then
 			info "Adding agent config for ${AA_KBC}"
-			AA_KBC_PARAMS="offline_sev_kbc::null" envsubst < "${script_dir}/agent-config.toml.in" | tee "${ROOTFS_DIR}/etc/agent-config.toml"
+			AA_KBC_PARAMS="${AA_KBC}::null" envsubst < "${script_dir}/agent-config.toml.in" | tee "${ROOTFS_DIR}/etc/agent-config.toml"
```

Comment:
Sorry to add this after previous approving, but I just wanted to check why this change was needed?
- If you want to set the `aa_kbc_params` we seem to have moved to using kernel_params as it's more flexible and updatable 
- If you want to restrict the endpoints I have some concerns about the brittleness of it and we've discussed this a little bit in https://github.com/kata-containers/kata-containers/issues/5590

Reply:
Yep, totally valid. I will resolve your concern. Thanks!